UPGRADING.md

```diff
@@ -69,6 +72,8 @@ if needed.
   $ tendermint gen_node_key > $TMHOME/config/node_key.json
   ```
 
+* The `--fast-sync` flag has been renamed to `--enable-block-sync`.
```

Comment:
Is this subsumed by the above? (Repeating it is fine if it's on purpose)

Reply:
Removed this reference to it!